web/src/components/HeaderTitle.tsx

```diff
@@ -1,44 +1,74 @@
 import styled from '@emotion/styled'
+import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown'
+import Typography from '@mui/material/Typography'
 import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
 
+import buildConfig from '../constants/buildConfig'
 import dimensions from '../constants/dimensions'
+import Link from './base/Link'
+import Tooltip from './base/Tooltip'
 
 const LONG_TITLE_LENGTH = 25
 export const HEADER_TITLE_HEIGHT = 50
 
-const HeaderTitleContainer = styled.div<{ long: boolean }>`
+const HeaderTitleContainer = styled(Typography)<{ withPadding?: boolean }>`
   display: flex;
-  align-items: center;
-  font-size: ${props => (props.long ? '1.3rem' : '1.8rem')};
+  align-items: flex-start;
   max-height: ${dimensions.headerHeightLarge};
-  font-weight: 800;
   flex: 1;
   order: 2;
-  padding: 0 10px;
-  box-sizing: border-box;
 
-  ${props => props.theme.breakpoints.up('lg')} {
-    font-size: ${props => (props.long ? '1.5rem' : '1.8rem')};
+  ${props => props.withPadding && props.theme.breakpoints.up('md')} {
+    padding-bottom: 8px;
```

Comment:
This padding only for integreat (it looks better when the title is on the same level with the integreat logo text).